pkg/kp/kv.go

```diff
@@ -37,6 +38,18 @@ func NewStore(opts Options) *Store {
 	return &Store{client: client}
 }
 
+// KVError encapsulates an error in a Store operation. Errors returned from the
+// Consul API cannot be exposed because they may contain the URL of the request,
+// which includes an ACL token as a query parameter.
```

Comment:
already done, went on the mailing list and armon opened https://github.com/hashicorp/consul/issues/791. But it might be a while before they get it fixed. If I was the one who had to fix it I'd probably do so grudgingly while grumbling about special cases for *that one* query parameter, so I can't blame them.